src/libs/ajax.js

```diff
@@ -873,7 +876,6 @@ const GoogleBilling = signal => ({
   }
 })
 
-
```

Comment:
gimme back my line

Reply:
gimme back my 2 lines

Reply:
sorry - misinterpreted this the other way